css-grid-classes/style.css

```diff
@@ -0,0 +1,45 @@
+.container {
+  max-width: 800px;
+  margin-left: auto;
+  margin-right: auto;
+  text-align: center;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  margin-bottom: 20px;
+}
+
+h2 {
+  height: 15%;
+}
+
+.column-link-yoshi {
+  font-size: 10px;
+  display: flex;
+  justify-content: space-evenly;
+}
+
+.text {
+  display: inline-block;
+  font-size: 15px;
+  font-weight: bold;
+}
+
+.column-mario-dk {
+  width: 800px;
```

Comment:
Rather than using a set pixel size like this, you should use a responsive unit such as `%`, since the container element has a set size you can work off of. Also, grid classes are not named after the elements they are used for, but after the functionality they offer. For instance, the functionality of this class is meant to be to take up half the available area, so a better name would be something like `column-half`